package.json

```diff
@@ -32,7 +32,6 @@
     "@trivago/prettier-plugin-sort-imports": "^3.2.0",
     "@typescript-eslint/eslint-plugin": "^5.13.0",
     "@typescript-eslint/parser": "^5.13.0",
-    "cross-env": "^7.0.3",
```

Comment:
@erikmejias This package is used for setting the `process.env.NODE_ENV` variable consistently in the npm scripts.

Reply:
I though on removing it since we dropped the `dot-env` package a few updates ago when switching to serving files locally.

Is there any other use for `cross-env`?

Reply:
Both packages are unrelated.
`cross-env` is used in the [npm scripts](https://github.com/finsweet/developer-starter/blob/4dbae9d9e88786ee10f8e245fc8ead867965c76c/package.json#L9-L10).

Reply:
It's used to set the `NODE_ENV` variable consistently across different OS (Windows, MAc, Linux)

Reply:
Oh right! My bad, don't know why I thought otherwise 🙈